bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableChannels.java

```diff
@@ -66,9 +66,9 @@
   private static final String POST_RETRY_REPORT_ENTRY = "PostRetry";
   /** Number of threads to use to initiate retry calls */
   private static final int RETRY_THREAD_COUNT = 5;
-  
+
   /** Number of seconds to wait for a termination before trying again. */
```

Comment:
We don't terminate the channel.  We awaitTermination for 5 seconds and then try again.